mains/5-veggies-mix

```diff
@@ -0,0 +1,10 @@
+# TITLE: 5 (five) veggies mix
```

Comment:
There appear to only be 3 different veggies!

Reply:
oh no, you are right! My mistake